src/main/java/yalter/mousetweaks/Main.java

```diff
@@ -310,13 +312,14 @@ private static void handleWheel(Slot selectedSlot) {
 
 		List<Slot> slots = handler.getSlots();
 
-		if (config.wheelScrollDirection == WheelScrollDirection.INVERTED || (config.wheelScrollDirection
-		                                                                     == WheelScrollDirection.INVENTORY_POSITION_AWARE
-		                                                                     && otherInventoryIsAbove(selectedSlot,
-		                                                                                              slots))) {
+		// this could be combined with !=, but it's more readable this way
```

Comment:
```suggestion
```
I really like the way you refactored this! But this comment doesn't belong in the code. 😀

Reply:
Personally, that's exactly the type of thing I'd use comments for, but it's your codebase after all :p